README.md

```diff
@@ -3,50 +3,76 @@
 [![Build Status](https://api.travis-ci.org/b9org/b9.svg?branch=master)](https://travis-ci.org/b9org/b9)
 [![Coverage Status](https://coveralls.io/repos/github/b9org/b9/badge.svg?branch=master)](https://coveralls.io/github/b9org/b9?branch=master)
 
-Base9 is an educational language runtime and programming language! We’re using it to show people how to use
-[Eclipse OMR] to build their own language runtime with a Just in Time (JIT) Compiler! The Base9 language, b9porcelain, is a simple subset of JavaScript.
+b9 is an educational language runtime and programming language! We’re using it to show people how to use
+[Eclipse OMR] to build their own language runtime with a Just in Time (JIT) Compiler! The b9 language, b9porcelain, is a simple subset of JavaScript.
 
 [Eclipse OMR]: https://github.com/eclipse/omr
 
-## Requirements
+This page contains basic instructions on building b9 in an Ubuntu 16.04 environment.
+You may copy each of the blocks of commands and paste them in the terminal.
+Depending on your user privileges, you may need to precede some of commands with
+sudo in steps 1 and 3. For more detailed instructions, check out our 
+[set-up page](./docs/SetupBase9.md).
 
-To get started with Base9, you'll need:
+### 1. Requirements
 
-`git, build-essential, nodejs, npm, esprima, cmake, ninja` 
+To get started with b9 on Ubuntu 16.04, you'll need the following:
 
-For detailed instructions on setting up Base9, check out our [set-up page](./docs/SetupBase9.md).
+* `git` 
+* `build-essential`
+* `nodejs`
+* `npm` 
+* `cmake` 
+* `ninja`
 
+To install the applications above, run the following command
+```plaintext
+apt-get update && apt-get install git build-essential nodejs npm cmake ninja
+```
 
-## Clone the repository and get the submodules
+### 2. Clone the repository and get the submodules
 
 ```
-git clone --recursive https://github.com/b9org/b9.git
-cd Base9
-git submodule update --init
+git clone --recursive https://github.com/b9org/b9.git \
+&& cd b9 \
+&& git submodule update --init \
+&& cd ..
+```
```

Comment:
These last three steps are a no-op. Cloning the repository only requires a `git clone`. Can you mark this code block as `sh`?